### DISCUSSION TITLE
Cannot install SFML!!

### DISCUSSION THREAD

**Discussion by Roh1t-Sharma** [Homebrew/discussions#4001] [Getting started]
I've been trying to install SFML on my MBA M1, tried using the "brew install sfml", but didn't worked. 
Following is what i'm getting. Its showing an error, dunno how to resolve it.
<img width="658" alt="Screenshot 2022-11-30 at 8 40 04 AM" src="https://user-images.githubusercontent.com/118717994/204716737-dabea092-fe2a-43f3-af0d-ea6cf3e5937f.png">

**Comment by SMillerDev** [comment]
That screenshot isn't showing what you said you ran. What does `brew doctor` say though?

**Reply by Roh1t-Sharma** [reply]
<img width="1016" alt="Screenshot 2022-11-30 at 1 53 38 PM" src="https://user-images.githubusercontent.com/118717994/204777735-4976dc9f-3323-40fd-b930-1b53abcbb910.png">
It's showing the same thing

**Reply by gromgit** [reply]
And what's the output of `brew doctor` (as @SMillerDev asked) and `brew config`?

**Reply by Roh1t-Sharma** [reply]
<img width="649" alt="Screenshot 2022-12-01 at 5 44 03 PM" src="https://user-images.githubusercontent.com/118717994/205082043-5632f858-ba77-4c47-ac09-e5a0a0fb66ee.png">

**Reply by Roh1t-Sharma** [reply]
<img width="716" alt="Screenshot 2022-12-01 at 5 45 06 PM" src="https://user-images.githubusercontent.com/118717994/205082331-d7725463-5b87-4547-a2c0-d66afa8e7e52.png">

**Reply by SMillerDev** [reply]
The very first warning in your `brew doctor` output will most likely fox your issue.

**Reply by Roh1t-Sharma** [reply]
<img width="1342" alt="Screenshot 2022-12-01 at 6 52 38 PM" src="https://user-images.githubusercontent.com/118717994/205098755-e2c33922-d288-4c29-b9e9-1ca24caca6aa.png">

**Reply by Roh1t-Sharma** [reply]
Yea, it did.

**Reply by chenrui333** [reply]
I think it works now?

**Reply by Roh1t-Sharma** [reply]
Yes, It works now. 
Thanks alot everyone.

**Comment by chenrui333** [comment]
Works for me. Can you also run `brew search sfml`? I am also on m1 machine

```
$ brew install sfml
==> Downloading https://ghcr.io/v2/homebrew/core/sfml/manifests/2.5.1_2
Already downloaded: /Users/rui/Library/Caches/Homebrew/downloads/b61be6383678ff6d5de65220e98abc9de96efece2168cddf02a7d1328925cddb--sfml-2.5.1_2.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/sfml/manifests/2.5.1_2
######################################################################## 100.0%
==> Downloading https://ghcr.io/v2/homebrew/core/sfml/blobs/sha256:ca56d2babc673b63418e28ff62d1
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:ca56d2babc
######################################################################## 100.0%
==> Reinstalling sfml
==> Pouring sfml--2.5.1_2.arm64_ventura.bottle.tar.gz
🍺  /opt/homebrew/Cellar/sfml/2.5.1_2: 634 files, 9.9MB
==> Running `brew cleanup sfml`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).

$ brew search sfml
==> Formulae
csfml                           sfml ✔                          shml
```

**Reply by Roh1t-Sharma** [reply]
<img width="361" alt="Screenshot 2022-12-01 at 1 03 19 PM" src="https://user-images.githubusercontent.com/118717994/205024430-f371e6e9-3572-4298-bb43-07bcdf937028.png">
